### DISCUSSION TITLE
Support for Carousel

### DISCUSSION THREAD

**Discussion by silicahd** [givanz/Vvveb#65] [Ideas]
I realized it works great if you have it in the final mode but there is no way to actually use it in the editor. 

Something Owl Carousel will be great 
https://www.jqueryscript.net/demo/Powerful-Customizable-jQuery-Carousel-Slider-OWL-Carousel/

**Comment by givanz** [comment]
Thanks for the suggestion, it was in development for a time but it wasn't finished.

It's now added in the last update https://github.com/givanz/Vvveb/commit/4714aca71cf8eeae12dae293d67edcc186e04d59 using swiper library, changed files in `public/js`.

You will need to clear cache after update.
